build.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+#
+# Setup nginx reverse proxy via Docker.
+#
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/resources/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+function _get_docker_image_id() {
+  echo "$(docker images -q reverse-proxy)"
+}
+
+function _get_docker_container_id() {
+  echo "$(docker ps -a -q --filter ancestor=reverse-proxy)"
+}
+
+function _stop_docker_container() {
```

Comment:
These don't need underscore prefixes because they are not "internal" like builder, because this is not an imported script.